py/torch_tensorrt/dynamo/conversion/impl/condition/ops.py

```diff
@@ -30,73 +28,30 @@ def where(
     x_shape = list(input.shape)
```

Comment:
Although this isn't related to the changes you made, isn't the check for broadcastability unnecessary since it is already verified by the torch operation?

Reply:
Yes I saw error in tensorrt when invalid input that is not boradcastable is passed. But maybe early failure could be helpful